src/dashboards/Compact/components/Tile/styles.scss

```diff
@@ -33,4 +33,12 @@
         align-items: center;
         font-size: 2rem;
     }
+
+    &__walking-time {
+        font-style: normal;
+        font-weight: normal;
+        font-size: 16px;
```

Comment:
Her er det fint om du bruker rem i staden for px. 16 px = 1 rem. Samme for line-height, 24px = 1.5rem.

Reply:
Godt innspill. Det skal jeg fikse